doc/radosgw/compression.rst

```diff
@@ -4,8 +4,9 @@ Compression
 
 .. versionadded:: Kraken
 
-The Ceph Object Gateway supports server-side compression of uploaded objects,
-using any of Ceph's existing compression plugins.
+The Ceph Object Gateway supports server-side compression of uploaded objects.
+This compression can be achieved with any of Ceph's existing compression
+plugins.
```

Comment:
The Ceph Object Gateway supports server-side compression of uploaded objects using any of the existing compression plugins.

Reply:
Fair enough. When we figure out why "make check" is failing, I'll update this.